automation/azureResources.py

```diff
@@ -0,0 +1,175 @@
+import os
+from haikunator import Haikunator
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.sql import SqlManagementClient
+
+REGION = 'westus2'
```

Comment:
hard coding variable values are not recommended. you can look at my code for asking parameters from the user